### ISSUE TITLE
[Input] components should have the same style as [Input] inside [Form] component

### ISSUE THREAD

**Issue by bytasv** [Semantic-Org/Semantic-UI#6253]
### Steps
I'm using react semantic-ui but I think it's relevant to style of the framework:
1. Restyle regular Input
2. Use that Input inside a Form

### Expected Result
I will be able to see restyled Input inside form

### Actual Result
Input looses style because From provides it's own style for Inputs

### Version
2.3.z

Why are there different styles for from inputs and standalone inputs? It seems that it forces me to duplicate restyling effort. Or is there something I don't get? This seems like a bug in the framework

**Comment by w96k** [comment]
Hey, can you provide some code that gets you confused? jsbin/jsfiddle demo would be nice

**Comment by bytasv** [comment]
@w96k let me give you a links to source of the variables it should be enough to see the problem:

[Input padding](https://github.com/Semantic-Org/Semantic-UI/blob/master/src/themes/default/elements/input.variables#L16)

[Form Input padding](https://github.com/Semantic-Org/Semantic-UI/blob/master/src/themes/default/collections/form.variables#L34)

If I went and changed the padding for `Input` component, I will have to do the same for `Form` `Input`.

Here is the screenshot for `Input`
<img width="1263" alt="screenshot 2018-03-08 22 57 40" src="https://user-images.githubusercontent.com/437214/37176277-92f2acb0-2324-11e8-8500-06b60cd6d6cd.png">


Here is the screenshot for `Form` `Input`
<img width="1280" alt="screenshot 2018-03-08 22 59 14" src="https://user-images.githubusercontent.com/437214/37176258-8080461e-2324-11e8-81fe-b82fdd9bc10b.png">

Now it's really strange why it is implemented like that? Why simple input is different from the form input? And how to deal with duplication if you want to change the style or if you want to extend it?

**Comment by hammy2899** [comment]
I agree this is inconsistent. I think this was implemented to add some flexibility to theming to allow the user to change the styles for both individually.

I think we should take some input from others to see what they think before changing anything.

**Comment by bytasv** [comment]
What I simply did for my own usecase as a temporary fix - I commented out form input declaration in `forms.less` file, now it seems to be working as expected. But I think it would be great to at least make it configurable so I don't have to mess up theme main files
